doc/source/ray-core/examples/batch_prediction.ipynb

```diff
@@ -0,0 +1,337 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Batch Prediction\n",
+    "\n",
+    "```{tip}\n",
+    "We storngly recommend using [Ray Datasets](https://docs.ray.io/en/latest/data/examples/nyc_taxi_basic_processing.html#parallel-batch-inference) or [AIR](https://docs.ray.io/en/latest/ray-air/predictors.html) to develop batch prediction, which will enable you to build it faster and more easily, and get the built-in benefits like auto-scaling actor pool.  Use Ray Core APIs directly only when that’s the only option for your use case. \n",
```

Comment:
Let's add a sentence to encourage them to file a GH issue if their use case cannot be satisfied by datasets so we get a chance to know their use case and know what we are lacking.